Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -251,7 +251,8 @@ private function _isAmountNumeric($fAmount)
      */
     private function _isPositiveBelowMax($fAmount, $fMaxAmount)
     {
-        return $fAmount > 0 && $fAmount <= $fMaxAmount;
+        return $fAmount > 0 && $fMaxAmount > 0 &&
```

Comment:
Please, add a short comment explaining what the `bcsub` calculation is used for.